cmd/kubeadm/app/util/config/initconfiguration_test.go

```diff
@@ -164,12 +161,12 @@ func TestInitConfigurationMarshallingFromFile(t *testing.T) {
 			in:          controlPlaneV1alpha3YAMLAbstracted,
 			expectedErr: true,
 		},
-		{ // v1beta1 -> internal
-			name:         "v1beta1ToInternal",
-			in:           controlPlaneV1beta1YAMLAbstracted,
-			out:          controlPlaneInternalYAMLAbstracted,
-			groupVersion: kubeadm.SchemeGroupVersion,
-		},
+		//{ // v1beta1 -> internal NB. test commented after changes required for upgrading to go v1.12
```

Comment:
@fabriziopandini let's move this comment to the top of the test (above `func TestInitConfigurationMarshallingFromFile...`) so it's more visible and remove the local case comment (as it does not do any difference).